### PR TITLE
Verbose callback and logging fixes

### PR DESCRIPTION
**Pull request by khardix**
Implementation of changes requested in sclorg#50.

### CODE REVIEW

File: tests/test_cli.py [hunk 1]
```diff
@@ -15,14 +13,16 @@
...
+
+
+@pytest.fixture
+def runner(root_logger):
     r = CliRunner()
-    try:
-        with r.isolated_filesystem():
-            yield r
-    finally:
-        # Make sure other tests are not affected by the ones with --verbose
```
Comment (khardix) [inline tests/test_cli.py:24]: This turns out to be not needed, as the callback is executed on every run, thus setting it for us.

File: tests/test_cli.py [hunk 2]
```diff
@@ -84,28 +84,14 @@ def test_parse_argv_no_options(tmpdir, option):
...
-    ctx = run.make_context('test_log_verbosity',
-                           options + recipe_arguments)
-    assert ctx.params['verbose'] is verbose
-
-    log = logging.getLogger()
-    # Initial state – if the test fails here, the app has changed
-    # and the test needs to be adjusted.
-    # The default log level is WARNING.
-    # https://docs.python.org/3/library/logging.html
-    assert log.getEffectiveLevel() == logging.WARNING
```
Comment (khardix) [inline tests/test_cli.py:99]: As the callback is run every time, there is no good way reset the initial state. Dropping this check should not harm anything.

File: rpmlb/cli.py [hunk 3]
```diff
@@ -17,6 +17,10 @@
 @click.option(
     '--verbose', '-v', is_flag=True, default=False,
     help='Turn on verbose logging.',
+    # Enable logging as early as possible
+    is_eager=True, expose_value=False,
+    callback=lambda ctx, param, verbose:
+        rpmlb_logging.configure_logging(verbose),
```
Comment (junaruga) [inline rpmlb/cli.py:23]: This `configure_logging` is called by both cases of with `--verbose` option and without `--verbose` option, right?

Comment (khardix) [reply rpmlb/cli.py:23]: Yes.

Comment (junaruga) [reply rpmlb/cli.py:23]: OK. I did not understand the logic before. That was a reason that I moved the callback logic to `run` function.

**Pull request merged**